routers/user/home.go

```diff
@@ -269,6 +266,19 @@ func Issues(ctx *context.Context) {
 	}
 	opts.LabelIDs = labelIDs
 
+	repoIDStrings := ctx.QueryStrings("repos[]")
```

Comment:
How about a standard http query array param `repos[0]=1&repos[1]=2`?

Reply:
I'm not sure I follow, sorry. This, in order to explicitly place the elements in certain positions of the array?

Reply:
@lunny Is this still relevant?

Reply:
I am resolving this as it is an outdated version of the code, if still relevant let me know and I will remediate.